src/LoginForm.test.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import LoginForm from './LoginForm';
+
+describe('LoginForm', () => {
+  const fields = {
+    email: '',
+    password: '',
+  };
+  const onChange = jest.fn();
+  const onSubmit = jest.fn();
```

Comment:
```suggestion
  const handleChange = jest.fn();
  const handleSubmit = jest.fn();
```

handle과 on의 차이에 주의해주세요.